sdk/dotnet/Pulumi/Deployment/DeploymentOutputs.cs

```diff
@@ -0,0 +1,48 @@
+using System;
```

Comment:
Copyright is missing. (Btw, don't we have a linter for this?)

Reply:
Added the copyright